elements/lisk-transactions/test/unlock_token.spec.ts

```diff
@@ -12,142 +12,145 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import * as validUnlockTransactionScenario from '../fixtures/unlock_transaction/unlock_transaction.json';
+// import * as validUnlockTransactionScenario from '../fixtures/unlock_transaction/unlock_transaction.json';
 
-import { unlockToken } from '../src/unlock_token';
-import { TransactionJSON } from '../src/types';
+// import { unlockToken } from '../src/unlock_token';
+// import { TransactionJSON } from '../src/types';
 
 describe('#unlockToken transaction', () => {
-	let unlockTokenTransaction: Partial<TransactionJSON>;
+	it('should be updated or deleted once unlockToken design is decided', () => {
```

Comment:
maybe it's better to use multi-line comment as its easier to follow the changes 😅